stream-process/src/main/java/com/dellemc/oe/process/StreamProcess.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ */
+package com.dellemc.oe.process;
+
+import java.net.URI;
+
+import io.pravega.client.admin.StreamManager;
+import io.pravega.client.stream.Stream;
+import io.pravega.connectors.flink.FlinkPravegaReader;
+import io.pravega.connectors.flink.PravegaConfig;
+import io.pravega.connectors.flink.serialization.PravegaSerialization;
+import org.apache.flink.streaming.api.datastream.DataStream;
+import org.apache.flink.streaming.api.environment.StreamExecutionEnvironment;
+
+import com.dellemc.oe.util.CommonParams;
+/**
+ * A simple example app that reads from an exisiting Pravega stream using the FlinkConnector
+ */
+public class StreamProcess {
+
+	
+	public static void main(String[] args) throws Exception {
+ 
+        final String scope = CommonParams.getScope();
```

Comment:
Flink apps should use ParameterTool to get configuration from the command line. This allows the Nautilus UI to be used to specify configuration.
See the example in https://github.com/pravega/video-samples/blob/master/flinkprocessor/src/main/java/io/pravega/example/flinkprocessor/StreamToConsoleJob.java.